ytypes/util_test.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ytypes
+
+import (
+	"testing"
+
+	"github.com/openconfig/goyang/pkg/yang"
+	"github.com/openconfig/ygot/ygot"
+)
+
+func TestDataSchemaTreesString(t *testing.T) {
+	containerWithLeafListSchema := &yang.Entry{
```

Comment:
Maybe I'm mis-parsing here, but this doesn't have any leaf-lists in it? So should this be `containerWithListSchema`?

Reply:
Yes, it should. Done.